test/vanilla/ignite-new.test.ts

```diff
@@ -42,8 +42,8 @@ describe("ignite new", () => {
     })
 
     it("should print expected text to std out", () => {
-      expect(result).toContain(`Using ignite-cli`)
-      expect(result).toContain(`Ignite CLI ignited ${APP_NAME}`)
```

Comment:
I think this line, or something equivalent, should be present, as it'll fail if Ignite fails to fully spin up an app and print the success messages.

Reply:
Ah! I knew that it was printing std out, but for some reason it did not click that we were looking for the success message

Reply:
Added here: [feat(ignite-new): add success message test](https://github.com/infinitered/ignite/pull/2119/commits/b5a16064861a24977523b5690ad20ac8c7400886)